core/common/src/main/java/com/example/common/util/IntUtil.kt

```
@@ -0,0 +1,16 @@
+package com.example.common.util
+
+import android.annotation.SuppressLint
+
+@SuppressLint("DefaultLocale")
+fun Int.formatCount(): String {
+    return when {
+        this >= 10_000 -> {
+            val value = this / 10_000.0
+            String.format("%.1f만", value)
```

detekt (1.23.8)

[warning] String.format("%.1f만", value) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)